Configuration/Eras/python/Era_Run2024_approxSiStripCluster.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_2024_cff import Run3_2024
+from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
+from Configuration.ProcessModifiers.trackingNoLoopers_cff import trackingNoLoopers
+
+Run3_2024_approxSiStripClusters = cms.ModifierChain(Run3_2024.copyAndExclude([trackingNoLoopers]),  approxSiStripClusters) 
```

Comment:
just for my education, why do you want to have looper tracking in this era?

Also, if this is meant as an era, shouldn't it be registered also at https://github.com/cms-sw/cmssw/blob/bec516a41f1665d2693725299198a1b9236eff68/Configuration/StandardSequences/python/Eras.py ?